src/main/java/ru/smirnovv/FindMax.java

```diff
@@ -43,13 +53,19 @@ public static void main(final String[] args) {
         }
     }
 
-    private void run() throws Exception {
-        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
+    /**
+     * Method, where read lines and print maximum numbers.
```

Comment:
Executes the program flow reading lines from the standard input, finding maximum numbers in them and printing the result to the standard output.